backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -423,6 +426,7 @@ module Expr =
     | ERecord (id, _, _)
     | EConstructor (id, _, _, _)
     | EFeatureFlag (id, _, _, _)
+    | EForbiddenExpr (id, _, _)
```

Comment:
Are we able to remove the EForbiddenExpr from all places? The intent is to prevent the creating of invalid pipe expressions using the type system, and EForbiddenExpr subverts that a bit.

Reply:
Agree, I remove it